styles.css

```diff
@@ -8,54 +8,106 @@ body {
 }
 
 header {
-    background-image: url(code_skills.jpg);
+    background-image: url(mobile_code_image.jpg);
+    background-position: center;
     background-repeat: no-repeat;
     background-size: cover;
+    position: relative;
     width: 100%;
-    height: 1080px;
+    height: 100vh;
 }
 
 .container {
     margin: 0 auto;
-    width: 1200px;
-    overflow: auto;
+    width: 80%;
 }
 
-ul {
+.nav-main {
     overflow: hidden;
+    /*width: 100%;*/
 }
 
 li {
-    float: right;
     list-style: none;
     margin-right: 10px;
+    text-align: right;
 }
 
 .nav-buttons a:link, .nav-buttons a:visited {
     color: white;
+    display: inline-block;
+    padding: 15px 15px 15px 15px;
     text-decoration: none;
-    text-align: center;
-    padding: 10px 10px;
 }
 
 .nav-buttons a:hover {
     color: black;
-    text-decoration: none;
-    text-align: center;
-    padding: 10px 0px 10px 0px;
     background-color: white;
     border-style: solid;
     border-color: white;
+    display: inline-block;
+    padding: 15px 15px 15px 15px;
+    text-decoration: none;
 }
 
 section {
-    height: 1080px;
+    height: 100vh;
+    padding: 0 20 0 20;
 }
 
 .image-icons {
     max-width: 150px;
 }
 
+.footer-links {
+    overflow: auto;
+    position: relative;
+    width: 100%;
+    height: 30vh;
+}
+
+/*Tablet */
+
+@media only screen and (min-width: 600px) {
+    body {
+        background-color: lightblue;
```

Comment:
color wants to be hex code or rgb (but i'm guessing you're using this for working out your media queries)

Reply:
thanks Jack, going to change this in my styling tasks